odoo/tools/config.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, fname=None):
         group.add_option("-c", "--config", dest="config", help="specify alternate config file")
         group.add_option("-s", "--save", action="store_true", dest="save", default=False,
                           help="save configuration to ~/.odoorc (or to ~/.openerp_serverrc if it exists)")
-        group.add_option("-i", "--init", dest="init", help="install one or more modules (comma-separated list, use \"all\" for all modules), requires -d")
+        group.add_option("-i", "--init", dest="init", help="install one or more modules (comma-separated list, use \"all\" for base module), requires -d")
```

Comment:
It could be ``install (update) one or ...`` depending on the result of https://github.com/odoo/odoo/pull/24234